src/biogeochem/CNPhenologyMod.F90

```diff
@@ -2452,8 +2470,22 @@ subroutine CNOffsetLitterfall (num_soilp, filter_soilp, &
 
             if (offset_counter(p) == dt) then
                t1 = 1.0_r8 / dt
-               leafc_to_litter(p)  = t1 * leafc(p)  + cpool_to_leafc(p)
+               !leafc_to_litter(p)  = t1 * leafc(p)  + cpool_to_leafc(p)
```

Comment:
I wondered the same thing initially, then saw that it looks like this has just been moved to line 2486, below.

An alternative approach that may be better would be to introduce crop-specific parameters on the parameter file that specify the fraction of leafc that goes to litter vs. something like biofuelc for each crop. Most crops could have this fraction be 0 (i.e., all to litter).

Reply:
I agree that it is better to put a cut fraction in the parameter file as a crop-specific parameter

Reply:
I agree with Bill it will be better to introduce a cut fraction in the parameter file as a crop-specific parameter.

Reply:
@danicalombardozzi is something like this what Mike is doing for residue removal?

Reply:
Yes, this is similar to what @mwgraham is doing for residue removal. I don't remember the exact details, but the residue management removes a large fraction of the leaf and stem residue, and it is put in the 1-year product pool. He does not currently include a PFT-level option, but I think setting up residue harvest as a CFT-level parameter will allow for more flexibility and realism so that different fractions of each crop can be harvested.